README.md

```diff
@@ -67,4 +67,4 @@ This project is in the worldwide [public domain](LICENSE.md). As stated in [CONT
 
 > This project is in the public domain within the United States, and copyright and related rights in the work worldwide are waived through the [CC0 1.0 Universal public domain dedication](https://creativecommons.org/publicdomain/zero/1.0/).
 >
-> All contributions to this project will be released under the CC0 dedication. By submitting a pull request, you are agreeing to comply with this waiver of copyright interest.
```

Comment:
Looks like this may be a stray change — nbd, but `git add -p` can help avoid that in the future. ✨

Reply:
This is my Sublime settings being aggressive about trimming trailing whitespace — I can turn that setting off to reduce whitespace noice. GitHub also has a "Hide Whitespace Changes" button for file review.